packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -3,23 +3,33 @@
 import React, { useState } from "react";
 import CustomXAxis from "./CustomXAxis";
 import { scaleSymlog } from "d3-scale";
-import { Area, CartesianGrid, ComposedChart, Line, ResponsiveContainer, Tooltip, XAxis, YAxis } from "recharts";
+import { Area, Bar, CartesianGrid, ComposedChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from "recharts";
 import { DataSet, ImpactVectors } from "~~/app/types/data";
 
 const logScale = scaleSymlog();
 
 const NON_VECTOR_KEYS = ["image", "name", "profile", "opAllocation"];
-const VECTOR_COLORS = [
-  "text-[#ffa500]",
-  "text-[#00ff00]",
-  "text-[#0000ff]",
-  "text-[#ffff00]",
-  "text-[#00ffff]",
-  "text-[#008000]",
-  "text-[#ff00ff]",
-  "text-[#800080]",
-  "text-[#000080]",
-];
+
+const COLOR_CLASS_BY_VECTOR: { [key in keyof ImpactVectors]: string } = {
+  "OSO: # GitHub Repos": "text-[#ff0000]",
+  "OSO: Date First Commit": "text-[#00ff00]",
+  "OSO: Total Stars": "text-[#0000ff]",
+  "OSO: Total Forks": "text-[#ffff00]",
+  "OSO: Total Contributors": "text-[#ff00ff]",
+  "OSO: Contributors Last 6 Months": "text-[#00ffff]",
+  "OSO: Avg Monthly Active Devs Last 6 Months": "text-[#ff8000]",
+  "OSO: # OP Contracts": "text-[#8000ff]",
+  "OSO: Date First Txn": "text-[#00ff80]",
+  "OSO: Total Onchain Users": "text-[#ff0080]",
+  "OSO: Onchain Users Last 6 Months": "text-[#80ff00]",
+  "OSO: Total Txns": "text-[#0080ff]",
+  "OSO: Total Txn Fees (ETH)": "text-[#ff0080]",
+  "OSO: Txn Fees Last 6 Months (ETH)": "text-[#ff80ff]",
+  "OSO: # NPM Packages": "text-[#80ff80]",
+  "OSO: Date First Download": "text-[#8080ff]",
+  "OSO: Total Downloads": "text-[#ffff80]",
+  "OSO: Downloads Last 6 Months": "text-[#80ffff]",
+};
```

Comment:
I chose these colors pretty randomly. We can change them to whatever else if anyone has an opinion

Reply:
Yeah, that is perfectly fine. We will get a designers opinion for the final colors.